packages/venia-ui/lib/components/WishlistPage/wishlist.js

```diff
@@ -17,36 +19,63 @@ import ActionMenu from './actionMenu';
  * @param {boolean} props.shouldRenderVisibilityToggle whether or not to render the visiblity toggle
  */
 const Wishlist = props => {
-    const { data, shouldRenderVisibilityToggle } = props;
+    const { data, shouldRenderVisibilityToggle, collapsed } = props;
```

Comment:
Can you add `collapsed` to the JS Docs? Also, if this is a boolean value, please called it `isCollapsed` instead.